BUILD.BAT

```diff
@@ -28,5 +30,4 @@ set z_ReturnCode=%ERRORLEVEL%
 echo %0 ERROR: Encountered RC %z_ReturnCode% - exiting
 :return
 popd
-rem ----- Lvl(%z_NestLevel%)  End %0 %1 %2 %3 %4 %5 %6 %7 %8 %9
```

Comment:
@adelosa could you please re-insert this comment line? (31 in the original):
rem ----- Lvl(%z_NestLevel%)  End %0 %1 %2 %3 %4 %5 %6 %7 %8 %9

Reply:
Fixed by cdcd848